.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - applications/**/*.json
+
+jobs:
+  parse:
+    name: Parse
+    runs-on: ubuntu-latest
+
+    outputs:
+      rdnn: ${{ steps.parse.outputs.rdnn }}
+      version: ${{ steps.parse.outputs.version }}
+      source: ${{ steps.parse.outputs.source }}
+      commit: ${{ steps.parse.outputs.commit }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Parse
+        id: parse
+        uses: elementary/action-appcenter-review-parse@main
```

Comment:
This action takes all of the modified files in the PR, finds the one that matches the regex given (`applications/<rdnn>/<version>.json`) and then parses that file and merges it with the regex to get the `rdnn`, `version` (git tag / semver version), `source` (any valid public git repo), and git `commit`. It then runs the next step which is our standard flatpak building step.